snuba/web/delete_query.py

```diff
@@ -27,23 +30,68 @@ def construct_condition(body: Dict[str, Any]) -> Expression:
     return combine_and_conditions(and_conditions)
 
 
+def _get_rows_to_delete(
+    storage: WritableTableStorage, select_query_to_count_rows: Query
+) -> int:
+    formatted_select_query_to_count_rows = format_query(select_query_to_count_rows)
+    select_query_results = (
```

Comment:
I wonder if it would be useful to use  `_format_storage_query_and_run` in here? I haven't looked at it too deeply but it would be nice to re-use error handling, retry, and logging logic  cc @volokluev

Reply:
I think this is fine as is. Unless there's something specific about the error handling that you want to call out